nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/test/java/org/apache/nifi/provenance/ITestPersistentProvenanceRepository.java

```diff
@@ -504,7 +485,6 @@ public void testIndexOnRolloverWithImmenseAttribute() throws IOException {
 
     @Test
     public void testIndexOnRolloverAndSubsequentSearch() throws IOException, InterruptedException, ParseException {
-        assumeFalse(isWindowsEnvironment());
```

Comment:
Shouldn't these have `@DisabledOnOs(OS.WINDOWS)` on them?

Reply:
I considered adding the annotation, but since this is an integration test, it is not run automatically. It looks like the test was originally a unit test that was changed to an integration test for performance reasons. Since it doesn't run automatically, and does not appear to have platform-specific operations, it seems better to leave off the `DisabledOnOs` annotation, but I can add it at the class level if you think it still makes sense to retain.

Reply:
I'm fine with leaving it as-is based on your response.